diff --git a/node_modules/@stripe/stripe-react-native/ios/StripeSwiftInterop.h b/node_modules/@stripe/stripe-react-native/ios/StripeSwiftInterop.h
index 44984e8..55b86e1 100644
--- a/node_modules/@stripe/stripe-react-native/ios/StripeSwiftInterop.h
+++ b/node_modules/@stripe/stripe-react-native/ios/StripeSwiftInterop.h
@@ -11,7 +11,7 @@
 @protocol STPPaymentCardTextFieldDelegate;
 @protocol STPCardFormViewDelegate;

-typedef NS_ENUM(NSUInteger, STPPaymentStatus);
+typedef NS_ENUM(NSInteger, STPPaymentStatus);

 // stripe_react_native-Swift.h also depends on these headers.
 #import <React/RCTBridgeModule.h>
